vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -140,7 +162,7 @@ private IVSVideo saveVideoWithUrl(String url, String title) {
         return videoRepo.save((VSVideo) vidContent);
     }
 
-    private IVSVideo saveVideo(byte[] video, Long size, String filename, String title) {
+    public IVSVideo saveVideo(byte[] video, Long size, String filename, String title) {
```

Comment:
is this method ever called outside this class (or is it supposed to be)? if not, it should not be public